pkg/ide/jetbrains.go

```diff
@@ -86,3 +94,30 @@ func isAlreadyDownloaded(projectHostname, downloadPath string) bool {
 	err := statCmd.Run()
 	return err == nil
 }
+
+func getJetbrainsVersion(productCode string) (string, error) {
+	jetbrainsDataServicesUrl := fmt.Sprintf("https://data.services.jetbrains.com/products/releases?code=%s&type=release&latest=true&build=", productCode)
+	res, err := http.Get(jetbrainsDataServicesUrl)
+	if err != nil {
+		return "", err
+	}
+
+	body, err := io.ReadAll(res.Body)
+	if err != nil {
+		panic("failed to read version for product")
+	}
+
+	var result map[string][]map[string]interface{}
+	if err := json.Unmarshal(body, &result); err != nil {
+		panic("failed to parse json for product")
+	}
+
+	for _, v := range result {
+		if len(v) > 0 {
+			if version, ok := v[0]["version"].(string); ok {
+				return version, nil
+			}
+		}
+	}
+	panic("no version found")
```

Comment:
@bnova-stefan again, this function should not panic. Please return an error here.